project/externals/aws-sdk-cpp.cmake

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2022 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License.
+
+if(ENABLE_ROCKSDB_CLOUD)
+    message(STATUS "use rocksdb cloud")
+    set(name aws-sdk-cpp)
+    set(source_dir ${CMAKE_CURRENT_BINARY_DIR}/${name}/source)
+
+    ExternalProject_Add_Git(
+        ${name}
+        GIT_REPOSITORY https://github.com/aws/aws-sdk-cpp.git
+        GIT_TAG f61fa33a65d4c37cc3f2069d558771b45196158c  # As of 2022/9/19
+        #ARCHIVE_FILE aws-sdk-2022-9-19.tar.gz
+        #ARCHIVE_MD5 bee5b1a24318547232be73dbf06dbcc3
+        PREFIX ${CMAKE_CURRENT_BINARY_DIR}/${name}
+        TMP_DIR ${BUILD_INFO_DIR}
+        STAMP_DIR ${BUILD_INFO_DIR}
+        DOWNLOAD_DIR ${DOWNLOAD_DIR}
+        #PATCH_COMMAND patch -p1 < ${CMAKE_SOURCE_DIR}/patches/${name}-2022-9-19.patch
+        SOURCE_DIR ${source_dir}
+        CMAKE_ARGS
+            ${common_cmake_args}
+            -DCMAKE_BUILD_TYPE=Release
+            -DBUILD_TESTS=OFF
+            -DENABLE_TESTING=OFF
```

Comment:
aws-sdk is too large. We don't need all of them. Maybe you can limit the libraries to the ones we need at this moment with:
'-DBUILD_ONLY=s3;kinesis;transfer'

Reply:
There is a problem with compiled packages when using the DBUILD_ONLY option.
Do not use this option for now